subtitles/zh-CN/31_navigating-the-model-hub.srt

```diff
@@ -305,12 +305,12 @@ that is relevant to the categories we have just seen.
 
 62
 00:03:01,320 --> 00:03:04,410
-文件和版本选项卡显示体系结构
+文件和版本选项卡显示

```

Comment:
Files & versions is a tag name on this page. Currently, HuggingFace’s webpage is still in English. I think it’s not necessary to translate it. It’s easier for readers to understand the page.e it. It’s easier for readers to understand the page.